docs/digger/admin/using-http-proxy.asciidoc

```diff
@@ -0,0 +1,44 @@
+== HTTP Proxy Support
+
+When you configure the Digger Installer to use proxy, it will automatically reconfigure the following components:
+
+
+* The Jenkins server
+* The Android SDK pod used to download Android SDKs
+* Container/node that will run the build jobs, by adding the appropriate environment variables like `HTTP_PROXY` and `HTTPS_PROXY` (where?)
```

Comment:
in the clusters inventory file - sample provided [here](https://github.com/aerogear/digger-installer/blob/master/inventory-sample)